fabric-api-base/src/main/java/net/fabricmc/fabric/api/event/EventFactory.java

```diff
@@ -47,6 +47,10 @@ public static void invalidate() {
 
 	/**
 	 * Create an "array-backed" Event instance.
+	 * The factory will be used to create the invoker for any number of listeners.
```

Comment:
This one is redundant imo. I don't see why people would think differently about how the invoker factory works. If there should be documentation, it should be on the old unintuitive version than this version.